e2e/playwright/library-actions/src/tests/library-actions.spec.ts

```diff
@@ -224,6 +224,7 @@ test.describe('Library actions ', () => {
       await myLibrariesPage.reload({ waitUntil: domContentLoadedString });
       await expect(libraryTable.getRowByName(user2Library2)).toBeVisible();
       await libraryTable.getRowByName(user2Library2).click();
+      await myLibrariesPage.page.waitForTimeout(1000);
```

Comment:
Not nice but didn't find another useful wait/action to make the tests pass.

Reply:
Okay and why do you need to wait or what do you need to wait for in that case? According to Playwright this shouldn't be used in production code https://playwright.dev/docs/api/class-page

Reply:
here when we do check file as Favourite from API it shows as marked as Favourite, but on UI it will take few sec to reflect same.  And I talk with Tom as well, he said  this will be normal API take little Time reflect this changes.  on UI manually we do things it works a per requirement so for now for automation we are good with wait condition.